beets/dbcore/queryparse.py

```diff
@@ -109,7 +113,10 @@ def construct_query_part(model_cls, prefixes, query_part):
             return query_class(pattern)
 
     key = key.lower()
-    return query_class(key.lower(), pattern, key in model_cls._fields)
+    if query_class is query.NoneQuery:
+        return query_class(key.lower(), pattern)
```

Comment:
Aha! That's because it only takes two arguments (and self):

    def __init__(self, field, fast=True):

But they're `field` and `fast`, not `field` and `pattern`. So we can just have that method ignore its `pattern` argument.